deployJenkinsfile

```diff
@@ -1,9 +1,9 @@
-@Library(value='iow-ecs-pipeline@master', changelog=false) _
+@Library(value='iow-ecs-pipeline@2.2.6', changelog=false) _
 
 pipeline {
     agent {
         node {
-            label 'team:iow'
+            label 'team:nhgf'
```

Comment:
I cannot speak to this particular change, but I assume there is an `nhgf` worker node in jenkins that this team is using.  Otherwise everything else looks good!

Reply:
Yes, I verified that with @ayan-usgs earlier today.